torch_tvm/operators.cpp

```diff
@@ -404,6 +405,51 @@ RegisterTVMOperator reg({
        n->index = 0;
        return tvm::relay::TupleGetItem(n);
      }},
+    {Symbol::fromQualString("aten::fbgemm_linear_int8_weight_fp32_activation"),
+     [](Node* node, tvm::Array<tvm::relay::Expr> inputs) {
+       // inputs[0]: data_fp32
+       // inputs[1]: weight_int8
+       // inputs[2]: packed_weight
+       // inputs[3]: weight_acc
+       // inputs[4]: weight_scale
+       // inputs[5]: weight_zp
+       // inputs[6]: bias
+       TORCH_CHECK(
+           inputs.size() == 7, "Given input size from quantized linear", inputs.size(), " inputs");
+       auto op_find_minmax = tvm::relay::Op::Get("nn.quantize_findminmax");
+       auto op_choose_q_params = tvm::relay::Op::Get("nn.choose_quantize_params");
+       auto op_data_q = tvm::relay::Op::Get("nn.quantize_data_int8_quantize");
+       auto op_data_deq = tvm::relay::Op::Get("nn.quantize_data_mm_dequantize");
```

Comment:
Should we make dequantize a separate relayop? I am wondering if we can fuse dequantize and quantize of the next layer somehow. Or figure out some way to fuse those?

Reply:
Oh the bias is not quantized? Maybe we can try to quantize bias and weight as a joint quantity? Something to think over.

Reply:
Hmm, this involves the batchnorm issue. Currently, it is following the dynamic quantization schema.

Reply:
> Should we make dequantize a separate relayop? I am wondering if we can fuse dequantize and quantize of the next layer somehow. Or figure out some way to fuse those?

Normally, I think what you are saying is more like static quantization. We can collapse the quant and dequant, since they are the same q parameters. Dequant is normally fused with the op.

Reply:
Aah, yes. You are right. But if we are doing dequant just to find the min max values and then chose quantization params and quantize it again then can there not be a way to find fp32 min max values without going though dequant step? Dequant step should not be changing which value is min and which is max?

Reply:
I didn't get your point. Basically, dequant in this context is for int32 activation to fp32 conversion.